plugins/unifying/lu-device-runtime.c

```diff
@@ -85,7 +86,6 @@ lu_device_runtime_open (LuDevice *device, GError **error)
 		}
 	}
 	if (release != 0xffff) {
-		g_autofree gchar *devid2 = NULL;
```

Comment:
This can stay in this scope now, right?

Reply:
Indeed, my bad!

Reply:
Hum, after a check this is supposed to be out of this scope, since we use it to generate the guid after the bootloader string is read.